Marlin/src/lcd/menu/menu_configuration.cpp

```diff
@@ -306,6 +310,12 @@ void menu_configuration() {
 
   MENU_ITEM(submenu, MSG_ADVANCED_SETTINGS, menu_advanced_settings);
 
+  #if ENABLED(BABYSTEP_ZPROBE_OFFSET)
+    MENU_ITEM(submenu, MSG_ZPROBE_ZOFFSET, lcd_babystep_zoffset);
+  #elif HAS_BED_PROBE
+    MENU_ITEM_EDIT(float52, MSG_ZPROBE_ZOFFSET, &zprobe_zoffset, Z_PROBE_OFFSET_RANGE_MIN, Z_PROBE_OFFSET_RANGE_MAX);
+  #endif
```

Comment:
These items are already accessible from Advanced Menu, Bed Leveling Menu, and Tune Menu.

Let's make these items only accessible from one of these places instead of putting them everywhere.